expression/builtin_math_test.go

```diff
@@ -115,32 +116,65 @@ func (s *testEvaluatorSuite) TestFloor(c *C) {
 
 	sc := s.ctx.GetSessionVars().StmtCtx
 	tmpIT := sc.IgnoreTruncate
-	sc.IgnoreTruncate = true
```

Comment:
The `expression` package uses one `Suite` to run all the cases. The context is shared between cases. If we remove reset action, it maybe affect other cases.

If my understand is correct, I think we should keep it.